crates/astria-sequencer/src/app.rs

```diff
@@ -733,6 +737,10 @@ impl App {
     }
 }
 
+/// Display wrapper for `HashMap<RollupId, Vec<Deposit>>`.
+#[derive(Debug, serde::Serialize)]
+struct DisplayDeposits<'a>(&'a HashMap<RollupId, Vec<Deposit>>);
```

Comment:
This is extra wrapper is not necessary since serde implements `Serialize for HashMap`.

However, this newtype wrapper might be useful in case you also add snapshots tests using inta (see for example composer and conductor which employ them).